src/contracts/AbstractARM.sol

```diff
@@ -598,26 +642,35 @@ abstract contract AbstractARM is OwnableOperable, ERC20Upgradeable {
     }
 
     /// @dev Calculate the available assets which is the assets in the ARM, external withdrawal queue,
-    /// less liquidity assets reserved for the ARM's withdrawal queue.
+    /// and active lending market, less liquidity assets reserved for the ARM's withdrawal queue.
     /// This does not exclude any accrued performance fees.
-    function _availableAssets() internal view returns (uint256) {
-        // Liquidity assets, eg WETH, are priced at 1.0
+    function _availableAssets() internal view returns (uint256 availableAssets, uint256 outstandingWithdrawals) {
+        // Liquidity assets, eg WETH, in the ARM and lending markets are priced at 1.0
         // Base assets, eg stETH, in the withdrawal queue are also priced at 1.0
         // Base assets, eg stETH, in the ARM are priced at the cross price which is a discounted price
         uint256 assets = IERC20(liquidityAsset).balanceOf(address(this)) + _externalWithdrawQueue()
             + IERC20(baseAsset).balanceOf(address(this)) * crossPrice / PRICE_SCALE;
 
-        // The amount of liquidity assets (WETH) that is still to be claimed in the withdrawal queue
-        uint256 outstandingWithdrawals = withdrawsQueued - withdrawsClaimed;
+        if (activeMarket != address(0)) {
```

Comment:
⛽️ `activeMarket` will be read from storage three times here in normal use. Gas savings to making it a temp variable.

Reply:
Good pickup. I'll change as `AbstractARM` will also be used for the Lido ARM on Ethereum